core/test/junit/saros/concurrent/SplitOperationTest.java

```diff
@@ -60,7 +60,7 @@ public void testInsertInsert() {
     assertEquals(Collections.singletonList(expected1), split1.toTextEdit(path, source));
 
     // vice versa
```

Comment:
I should replace this comment with an actual description of the setup.

Reply:
I guess adding the current "layout"

```
  position  0 1 2 3 4 5 6 7 8 9
  ------------------------------
  start:   "0 1 2 3 4 5 6 7 8 9"
  ins1:    "0 1 2 3 4 c d 5 6 7" (8 and 9 shifted out of view)
  ins2:    "0 1 2 a b 3 4 c d 5" (6 and 7 shifted out of view)
```

Either as comment above the test or before every operation transformation and assertion could be really helpful to understand what is tested here without digging through the whole code or papers regarding Jupiter.

Reply:
Yes, I could do that. But doing this for every test setup gets very tedious, especially with the later ones using multiple split operations. Furthermore, I will have to extend this test suite as part of my upcoming refactoring of the text edit position handling. The new test cases will contain more complex setups (multi-line changes, etc.), which makes this documentation style not feasible.

I will adjust the class javadoc for the test class to better indicate what is being tested. With this information, the short description of initial activities and the resulting (expected) outcome should be descriptive enough in my opinion.

Would that be an acceptable solution for you?

Reply:
As you are going to rewrite the test cases anyway I do not think that the documentation has to be included / updated with this patch.